ledger-service/http-json/src/it/scala/http/HttpServiceIntegrationTestUserManagement.scala

```diff
@@ -74,6 +74,22 @@ class HttpServiceIntegrationTestUserManagementNoAuth
 
   import scalaz.syntax.tag._
 
+  "Json format of UserRight should be stable" in Future {
+    import spray.json._
+    val ham = getUniqueParty("ham")
+    val spam = getUniqueParty("spam")
+    List[domain.UserRight](
+      domain.CanActAs(ham),
+      domain.CanReadAs(spam),
+      domain.ParticipantAdmin,
+    ).toJson shouldBe
+      JsArray(
+        JsObject("type" -> JsString("CanActAs"), "party" -> JsString(ham.unwrap)),
+        JsObject("type" -> JsString("CanReadAs"), "party" -> JsString(spam.unwrap)),
+        JsObject("type" -> JsString("ParticipantAdmin")),
+      )
```

Comment:
```suggestion
      List(
        Map("type" -> "CanActAs", "party" -> ham.unwrap),
        Map("type" -> "CanReadAs", "party" -> spam.unwrap),
        Map("type" -> "ParticipantAdmin"),
      ).toJson
```

Reply:
What would I do without you :facepalm: Completely oversaw that approach, many thanks! :smile: